packages/third-party/tailwind-preset/src/__tests__/plugins/soloTranslate.test.ts

```diff
@@ -0,0 +1,57 @@
+// Copyright 2025 The Lynx Authors. All rights reserved.
+// Licensed under the Apache License Version 2.0 that can be found in the
+// LICENSE file in the root directory of this source tree.
+import { describe, expect, it, vi } from 'vitest';
+
+import { soloTranslate } from '../../plugins/lynx/soloTranslate.js';
+import { runPlugin } from '../utils/run-plugin.js';
+
+describe('soloTranslate plugin', () => {
+  it('covers all branches', () => {
+    const { api } = runPlugin(soloTranslate, {
+      theme: {
+        translate: {
+          '1': '1rem',
+          '40px': '40px',
+          full: '100%',
+        },
+      },
+    });
+
+    const matchUtilities = vi.mocked(api.matchUtilities);
+
+    // Build a strictly typed utility map
+    type UtilityFn = (value: unknown) => Record<string, string> | null;
+
+    const utils = matchUtilities.mock.calls.reduce<Record<string, UtilityFn>>(
+      (acc, call) => {
+        const group = call[0] as Record<string, UtilityFn>;
+        for (const key in group) {
+          if (group[key]) {
+            acc[key] = group[key]!;
+          }
+        }
+        return acc;
+      },
+      {},
+    );
+
+    expect(utils['solo-translate-x']?.(0)).toBeNull();
+
+    expect(utils['solo-translate-x']?.('1rem')).toEqual({
+      transform: `translateX(1rem)`,
+    });
+
+    expect(utils['solo-translate-y']?.('1rem')).toEqual({
+      transform: `translateY(1rem)`,
+    });
```

Comment:
What will happen if user uses `solo-translate-x` and `solo-translate-y` simultaneously? In web, the latter style will override the previous so only `translateY` will take effects finally.